web/src/app/monitor/workbasket/monitor-workbasket-query-switcher/monitor-workbasket-query-switcher.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {MonitorQueryType} from '../../models/monitor-query-type';
+
+@Component({
+  selector: 'taskana-monitor-workbasket-query-switcher',
+  templateUrl: './monitor-workbasket-query-switcher.component.html',
+  styleUrls: ['./monitor-workbasket-query-switcher.component.scss']
+})
+export class MonitorWorkbasketQuerySwitcherComponent implements OnInit {
+
+  @Output()
+  queryChanged = new EventEmitter<MonitorQueryType>();
+  selectedChartType: MonitorQueryType;
+  monitorQueryPlannedDateType = MonitorQueryType.PlannedDate;
+  monitorQueryDueDateType = MonitorQueryType.DueDate;
+
+  constructor() {
+  }
+
+  ngOnInit() {
+    this.selectedChartType = MonitorQueryType.DueDate;
+    this.queryChanged.emit(MonitorQueryType.DueDate);
+  }
+
+  switch(queryType: MonitorQueryType) {
```

Comment:
This switch causes a refresh of the entire report. This is currenlty acceptable. Running in a production environment this will lead to a performance issue. - Maybe we can cache the report?